recipe/meta.yaml

```diff
@@ -55,6 +56,7 @@ outputs:
         #- prrte
         - libhwloc
         - libevent
+        - libnl  # [linux]
```

Comment:
Nit: I think all three of them have run_exports set up, so we don't need to add them manually.
```suggestion
```

Reply:
I do not understand your suggestion. This is the `run` section, not the `run_constrained` one. We definitely need libnl as a runtime dependency.

Reply:
The actual problem is: I do not understand this stuff in full, and with all the other related and unrelated mess, I get very very frustrated.

Reply:
"run_exports" is used to ensure ABI compatibility by conda-build. If a package is run_export'd, it means when used in the host section, conda-build will also add it to the run section, with proper pinning constraint set up. So we don't have to worry about it and let the upstream recipe maintainers (in this case, libnl et al) decide the pinning for us.

Reply:
If a package is not run_export'd, what we do here is wrong. We'd need to use pin_compatible to declare the pinning manually ourselves. It cannot be free dependencies without constraints.